lib/tasks_private/instance_types.rake

```diff
@@ -5,9 +5,6 @@ task :instance_types => [:environment] do
   # This task creates a instance_types.rb file in the current directory based upon the current
   # instance_types.rb file and data from
   #  curl https://raw.githubusercontent.com/powdahound/ec2instances.info/master/www/instances.json > lib/tasks_private/instance_types_data/instances.json
-  #  https://aws.amazon.com/ec2/instance-types/#instance-type-matrix > lib/tasks_private/instance_types_data/instance_types.csv
-  #  https://aws.amazon.com/ec2/previous-generation/#Previous_Generation_Instance_Details_and_Pricing_ > lib/tasks_private/instance_types_data/instance_types_previous.csv
```

Comment:
but this isnt included...
did you remove it, because we assume that previous generations do not change anymore?

Reply:
The original output from `ec2instance.info` already has a field `generation` which has value either 'current' or 'previous'. So that's how I determine it.